### PR TITLE
Import common files into this repo.

### PR DESCRIPTION
**Pull request by geeknoid**
- Delete circleci config as its completely redundant with what prow is doing.

- Delete other cruft

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,19 +0,0 @@
-workspace(name = "io_istio_api")
```
Comment (rshriram) [inline WORKSPACE]: can this be deleted? doesn't istio proxy depend on this?

**Comment by kyessenov** [comment]
You cannot remove bazel stuff or you're going to break proxy bazel build.

File: scripts/run_golangci.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ROOTDIR=$(dirname "${SCRIPTPATH}")
+cd "${ROOTDIR}"
+
+if [[ "$1" == "--fix" ]]
+then
+    FIX="--fix"
+fi
+
+# if you want to update this version, also change the version number in .golangci.yml
+GOLANGCI_VERSION="v1.16.0"
```
Comment (kyessenov) [inline scripts/run_golangci.sh]: why do we need to run golangci here? it automatically skips generated files, so it's entirely a no-op.

Comment (geeknoid) [reply scripts/run_golangci.sh]: This is a common file, copied from the istio/common-files repo.

**Comment by geeknoid** [comment]
OK, I restored WORKSPACE and BUILD. We should fix the proxy repo so that these turds aren't needed in istio/api...

**Pull request merged**